### ISSUE TITLE
Make award `title` field optional

### ISSUE THREAD

**Issue by tmorrell** [inveniosoftware/invenio-rdm-records#910]
## Is your feature request related to a problem? Please describe.
I'm migrating data from a system where only award numbers were required, and I don't have an automated way to get grant titles. When funding is added to the deposit form, it will also be annoying for users to have to enter a grant title for grants not in OpenAIRE.

## Describe the solution you'd like
Make the award 'title' an optional field.

## Describe alternatives you've considered
Ideally we would be able to automatically get the grant title from OpenAIRE, but we have funding from a number of sources that won't be indexed in OpenAIRE. Requiring grant titles is too large a burden for either researchers or library staff.

**Comment by lnielsen** [comment]
Ping @slint - I think you need to take this into account on the funder/grant sprint.

**Comment by ppanero** [comment]
Would it be an acceptabe solution to make it in a similar way than with the new *names* vocabulary? i.e.:
- If the grant exists in the vocabulary it will forcefully have the name (I guess is what Tom means by "automatically get the title from OpenAIRE").
- If the grant is not in the system (do we allow this??), then title would be optional.

**Comment by lnielsen** [comment]
I think what Tom is referring to is the record data model:

![Screenshot 2022-03-04 at 14 13 37](https://user-images.githubusercontent.com/1698163/156769761-a7ea0a8d-bf3f-44b4-bc49-4b64f2f8544c.png)

So both in the RDM record data model, and the grant vocabulary we cannot require ``title`` is set. So probably we should simply require that title **or** number is set.

**Comment by ppanero** [comment]
sorry, I didn't explain myself well. That's what I meant with *then title would be optional* in the second point. 👌

**Comment by tmorrell** [comment]
Yup, this issue is specific to the data model, as I'm loading in records with the API. title **or** number, or just requiring number would work fine.

I think reusing the names vocabulary is a very good approach for the funder/grant implementation.
